locale/en/blog/community/2017-election.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Node.js Foundation Individual Membership Director election opens Node.js opens Friday, January 20
```

Comment:
uuugh there's something weird with rendering on the page right now for me. I had removed this properly. I'll pull it up locally instead of using the GH UI.